eng/scripts/Update-PackageVersion.ps1

```diff
@@ -33,7 +33,7 @@ Param (
   [string] $PackageName,
   [string] $NewVersionString,
   [string] $ReleaseDate,
-  [boolean] $ReplaceLatestEntryTitle = $true
+  [boolean] $ReplaceLatestEntryTitle = $false
```

Comment:
1. `$false` is already default so this is unnecessary.
2. Why does engsys use `[boolean]`? You should use `[switch]` which not only works like a boolean, but you can easily specify as `-ReplaceLatestEntryTitle` without a value, which treats it as `$true`. You can still pass explicit values if you need by passing `:{value}` e.g. `-ReplaceLatestEntryTitle:$false`.

/cc @weshaggard since I see this often. It's not idiomatic to PowerShell and makes calling scripts harder for interactive use because users have to pass a value, which isn't idiomatic. Do you pass a value to `-Verbose`? To `-Force`? etc.

Reply:
1. sure
2. This script is only expected to be called by other scripts or from yaml. Having a different syntax for one parameter is odd.

```ps
  & "$EngDir/scripts/Update-PackageVersion.ps1" -ServiceDirectory $ServiceDirectory `
    -PackageName $PackageName `
    -NewVersionString $Version `
    -ReleaseDate $ReleaseDate `
    -ReplaceLatestEntryTitle:$ReplaceLatestEntryTitle
```